create_git_repository.md

```diff
@@ -8,4 +8,12 @@ To create a new GitHub project you can use the tool you prefer but it should hav
 * Should have `[project-name]` as a name
 * Should be private (unless you are creating an OpenSource project)
 
-Use the command `hub create renuo/[project-name]` to create the repo and add it to the origin of the current folder.
\ No newline at end of file
+Use the command `hub create renuo/[project-name]` to create the repo and add it to the origin of the current folder.
+
+However, if your repository is public, ensure that it contains a license. We usually use the [MIT](https://choosealicense.com/licenses/mit/) license if possible or a [CreativeCommons](https://creativecommons.org/licenses/) license for documentation-only repositories (such as the application setup guide).
+You can add a license template directly on GitHub:
```

Comment:
Can we follow those steps in this moment of the guide? No code commited yet. Did you try it out?